src/core/server/logging/layouts/json_layout.ts

```diff
@@ -54,7 +67,7 @@ export class JsonLayout implements Layout {
         pid: record.pid,
       },
     };
-    const output = record.meta ? merge(log, record.meta) : log;
+    const output = record.meta ? JsonLayout.mergeLogMeta(log, record.meta) : log;
```

Comment:
instead of manual filtering reserved paths out. we can always prioritize the `log` fields:
```js
const output = record.meta ? merge({ ...record.meta }, log) : log;
```

Reply:
Good point, there were only a few fields which I wasn't already overriding with core values. It will make it simpler to reason about if we just know that core's values will always "win" if duplicates are provided in the meta. Updated.